awestruct.gemspec

```diff
@@ -31,7 +31,6 @@ spec = Gem::Specification.new do |s|
     s.add_dependency 'bootstrap-sass', '>= 2.3.1.0'
     s.add_dependency 'zurb-foundation', '>= 4.0.9'
     s.add_dependency 'rest-client', '>= 1.6.7'
-    s.add_dependency 'ruby-s3cmd', '~> 0.1.5'
```

Comment:
I think we should add a note here, like what Haml does, and make this an optional dependency.